python_modules/libraries/dagster-dg-cli/dagster_dg_cli/api_layer/api/asset.py

```diff
@@ -21,17 +23,34 @@ def list_assets(
         self,
         limit: Optional[int] = 50,
         cursor: Optional[str] = None,
+        view: Optional[str] = None,
     ) -> "DgApiAssetList":
-        """List assets with cursor-based pagination."""
+        """List assets with cursor-based pagination and optional view."""
         # Enforce max limit constraint
         if limit and limit > 1000:
```

Comment:
Consider using a named constant or making this configurable.

Reply:
good idea